### ISSUE TITLE
Better icons

### ISSUE THREAD

**Issue by uruvelakassapa** [bkhpanigha/hh-suttas#131]
Perhaps we can move the icons above the search bar and also label them so it's clear what they stand for.
<img width="723" alt="Screenshot 2024-08-21 at 20 41 50" src="https://github.com/user-attachments/assets/2bbbc11d-f149-4144-9cd8-4a1b18706f85">.

~Also a nice to have would be abstract the html into one file which we can import into each of the other htmls so we dont need to manually copy and paste and ensure everything is consistent between pages~ done in #210

**Comment by uruvelakassapa** [comment]
@interested-in-dhamma yes I'm planning to move dark mode toggling to a settings panel. Hopefully it's out soon.

**Comment by Reptilioo** [comment]
Well, I just put titles on the buttons but feel free to add something more dynamic and prettier, I will do it eventually if no one has done it before

**Comment by Reptilioo** [comment]
Icons are moved above the search bar and have titles when hovering them, to show where they go.

Concerning this issue there is still:
- [ ] Nice to have: abstract the html into one file which we can import into each of the other htmls so we dont need to manually copy and paste and ensure everything is consistent between pages
- [ ] Nice to have: using something else than titles to appear when hovering the buttons, so it appears quicker and is customizable to be prettier.
